faust/tables/sets.py

```diff
@@ -84,7 +84,7 @@ def on_change(self, added: Set[VT], removed: Set[VT]) -> None:
         self.manager.send_changelog_event(self.key, OPERATION_UPDATE, [added, removed])
 
     def sync_from_storage(self, value: Any) -> None:
-        self.data = cast(Set, value)
+        self.data = set(value)
```

Comment:
I've always been baffled by the usage of `typing.cast` sprinkled all over, but it could just be remnants of the project from older versions of Python where typing was still a WIP.